packages/components/tooltip/README.md

```diff
@@ -25,6 +25,21 @@
 		</a>
 </p>
 
+## ➤ Properties
+
+| Property | Attribute | Type    | Default       | Description                                  |
+| -------- | --------- | ------- | ------------- | -------------------------------------------- |
+| tootlip  | tooltip   | string  | ''            | Tooltip text, simple string only.            |
+| position | position  | string  | right         | 'top', 'bottom', 'right', 'left'             |
+| width    | width     | string  | 'max-content' | Determining icon color, ex: 'error', 'focus' |
```

Comment:
Please, update the description :)

Reply:
Crap! Again used the same session of table generator :D Good catch 👍